cloud/linode/node_controller.go

```diff
@@ -57,7 +57,7 @@ func newNodeController(kubeclient kubernetes.Interface, client client.Client, in
 }
 
 func (s *nodeController) Run(stopCh <-chan struct{}) {
-	s.informer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+	_, err := s.informer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
```

Comment:
nit: again you can do the same contraction here

Reply:
Done, thanks :)